src/vaults/VaultRouter.sol

```diff
@@ -36,18 +37,20 @@ contract VaultRouter is Auth, BatchedMulticall, Recoverable, IVaultRouter {
     uint256 private constant REQUEST_ID = 0;
 
     ISpoke public immutable spoke;
+    IVaultRegistry public immutable vaultRegistry;
```

Comment:
I suppose we need to deploy a new VaultRouter anyways when we upgrade the VaultRegistry?